### PR TITLE
[Minor] Fixing trivy scan issue in pipeline.

### PR DESCRIPTION
**Pull request by TelicentPaul**

### CODE REVIEW

File: scg-docker/Dockerfile [hunk 1]
```diff
@@ -25,23 +25,24 @@ ARG LIB=${FUSEKI_DIR}/lib
...
 COPY ${JARS} lib
-ARG AGENTS=target/agents
+ARG AGENTS=scg-docker/target/agents
 COPY ${AGENTS} agents
 ARG PROJECT_VERSION
-ARG SBOM=target/docker-${PROJECT_VERSION}-bom.json
+ARG SBOM=scg-docker/target/docker-${PROJECT_VERSION}-bom.json
 COPY ${SBOM} /opt/telicent/sbom/smart-cache-graph-${PROJECT_VERSION}-bom.json
 
-ADD logback.xml .
```
Comment (rvesse) [inline scg-docker/Dockerfile:44]: Why is this removed?

Comment (TelicentPaul) [reply scg-docker/Dockerfile:44]: Not removed. Moved up to line 28 alongside the addition of entrypoint.sh

**Review by rvesse — APPROVED** [review]
